### DISCUSSION TITLE
1.5.0.4

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#490] [Обновления]
## Список изменений
- Добавлена поддержка coursehunter
- Улучшена очистка ссылок от @SashaXser в #472 

## What's Changed
* Более удачный regex для ссылок by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/472

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.0.3...1.5.0.4

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.0.4'>1.5.0.4</a>.</em>

**Comment by NullVerdict** [comment]
А давно ли этот баг присутствует? При переключении видео на ютубе, язык скидывается до en а потом уже определяется заново, это иногда может создать сообщение "Перевод видео займет". Такой баг вроде есть только на ютубе. 
Воспроизвести можно если часто переключать видео.
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/1b937e9a-2947-45df-8758-c086cc616d03)
